index.html

```diff
@@ -89,7 +89,8 @@ <h1>What's in Standard?</h1>
             <div v-for="round in truncate(rounds(undropped(sets)))" class="col mb-4">
               <div class="row row-cols-1 row-cols-sm-2 border mx-0 py-3 h-100">
                 <div class="col my-3" v-for="set in round">
-                  <a class="text-center p-3 mb-4 card text-dark h-100 justify-content-between" :class="{'bg-light': isReleased(set), 'text-muted border-0': !isReleased(set)}" style="min-height: 5rem"
+                  <!-- ja;lksdjfl;kasdjfljasdlfjl;sda -->
```

Comment:
?

Reply:
Oh, that I forgot to get rid of, I think I just needed to "bookmark" something to see when quickly scrolling by. Sorry lol